src/components/FeatureSummary.module.scss

```diff
@@ -1,10 +1,27 @@
+@import "../styles/variables";
+
+$font-size: 1.5rem;
+
 .feature-summary {
   display: flex;
   flex-direction: column;
   justify-content: center;
+  align-items: center;
   margin: 0 2rem;
+  height: 300px;
+
+  > p:nth-child(1) {
+    font-size: $font-size;
+    font-weight: $font-weight-extra-light;
```

Comment:
With the approach of text "weight" and "size" variables, you are forcing the developers to remember a specific combination. Try to use a mixin with the correct sets of combinations to be used in the whole app.

Reply:
What name is appropriate for the mixin?

Reply:
Up to you :)